metrics/New_Contributor_Closing_Issues.md

```diff
@@ -1,27 +1,28 @@
 # New Contributors Closing Issues
 
-Question: How many contributors are closing issues for the first time?
+Question: How many contributors are closing issues for the first time in a given project?
 
 ## Description
-This metric is an indication of the volume of contributors who are closing issues for their first time. When a contributor closes an issue for the first time it is some indication of "stickiness" for the individual within a project, especially for contributors who are not also committers.
+This metric is an indication of the volume of contributors who are closing issues for their first time within a given project. When a contributor closes an issue for the first time it is some indication of "stickiness" of the individual within that project, especially for contributors who are not also committers.
 
 ## Objectives
 To know how contributors are moving through the [contributor funnel](https://mikemcquaid.com/2018/08/14/the-open-source-contributor-funnel-why-people-dont-contribute-to-your-open-source-project/) by identifying “closing issues” as a milestone in the contributor journey.
 
 ## Implementation
 
 **Aggregators:**
-* Count. Total number of new contributors closing issues on this project for the first time during a given time period.
+* Count. Total number of contributors closing issues on this project for the first time during a given time period.
+* Percentage. Proportion of contributors closing issues on this project *for the first time* during a given time period, computed against *all* contributors having closed issues on this project during the same time period.
```

Comment:
+1 for aggregator, see comment above for language change.

Reply:
See earlier response.